openverse_catalog/dags/common/extensions.py

```diff
@@ -0,0 +1,17 @@
+from typing import Literal
+
+
+EXTENSIONS = {
+    "image": {"jpg", "jpeg", "png", "gif", "bmp", "webp", "tiff", "tif", "svg", "webp"},
+    "audio": {"mp3", "ogg", "wav", "aiff", "flac", "aac", "m4a", "wma", "mp4", "m4b"},
+}
+
+
+def extract_filetype(url: str, media_type: Literal["image", "audio"]) -> str | None:
```

Comment:
This looks great! Do you mind adding tests for these specific functions? Or do those tests exist in other PRs?